### ISSUE TITLE
AuthToken.reset()

### ISSUE THREAD

**Issue by mitchellporter** [ello/ello-ios#79]
Slightly confused on how `AuthToken's` reset function works:

```
static func reset() {
        var keychain = sharedKeychain
        keychain.authToken = nil
        keychain.refreshAuthToken = nil
        keychain.authTokenType = nil
        keychain.isPasswordBased = false
        keychain.username = nil
        keychain.password = nil
    }
```

If you look at the setters for the properties of `ElloKeychain`, they all perform checks for and prevent nil values from being set. Obviously you can nil a keychain value, but I'm failing to see how the design of `ElloKeychain` allows for this.

**Comment by colinta** [comment]
Oh crap, that's right!  I'll get that fixed.  Thanks for spotting that, and pointing it out.

**Comment by colinta** [comment]
Fixed in #81
